### PR TITLE
Upgrade to v1.75.0 of the AliCloud Terraform Provider

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: examples/examples_test.go [hunk 1]
```diff
@@ -34,6 +53,36 @@ func getJSBaseOptions(t *testing.T) integration.ProgramTestOptions {
 	return baseJS
 }
 
+func getCsharpBaseOptions(t *testing.T) integration.ProgramTestOptions {
+	region := getRegion(t)
+	base := getBaseOptions()
+	baseJS := base.With(integration.ProgramTestOptions{
```
Comment (mikhailshilkov) [inline examples/examples_test.go]: Nit
```suggestion
	baseDotNet := base.With(integration.ProgramTestOptions{
```

File: examples/examples_test.go [hunk 2]
```diff
@@ -34,6 +53,36 @@ func getJSBaseOptions(t *testing.T) integration.ProgramTestOptions {
...
+		Config: map[string]string{
+			"alicloud:region": region,
+		},
+		Dependencies: []string{
+			"Pulumi.AliCloud",
+		},
+	})
+
+	return baseJS
```
Comment (mikhailshilkov) [inline examples/examples_test.go]:
```suggestion
	return baseDotNet
```

File: examples/examples_test.go [hunk 3]
```diff
@@ -34,6 +53,36 @@ func getJSBaseOptions(t *testing.T) integration.ProgramTestOptions {
...
+}
+
+func getPythonBaseOptions(t *testing.T) integration.ProgramTestOptions {
+	region := getRegion(t)
+	base := getBaseOptions()
+	baseJS := base.With(integration.ProgramTestOptions{
```
Comment (mikhailshilkov) [inline examples/examples_test.go]: Nit: same as .NET. Or maybe just return without naming.

File: examples/examples_test.go (hunk 2 above)
Comment (stack72) [reply examples/examples_test.go]: fixed :)

File: examples/examples_test.go (hunk 1 above)
Comment (stack72) [reply examples/examples_test.go]: Fixed :)

File: examples/examples_test.go (hunk 3 above)
Comment (stack72) [reply examples/examples_test.go]: Fixed :)

**Pull request merged**